.github/workflows/on-pull-request-commit.yml

```diff
@@ -438,6 +453,93 @@ jobs:
           follow-up-finalization-check: true
         timeout-minutes: 10
 
+  run-e2e-committee-split-reserved-01:
+    needs: [build-test-node-image-and-e2e-client-image]
+    name: Run committee split test with node-0 and node-1 dead
```

Comment:
Are the roles of these nodes different? Te description doesn't really tell me what the test tests and why it needs three variants.